OWNERS_ALIASES

```diff
@@ -131,6 +131,7 @@ aliases:
     - prestist
     - marmijo
     - RishabhSaini
+    - c4rt0
```

Comment:
Could you please add it in alphabetical order?

Reply:
Thank you for this suggestion! Changes implemented.